Automattic/newspack-listings
# fix: remove unneeded editor CSS for Event Dates block

### All Submissions:






### Changes proposed in this Pull Request:



Looks like the markup and/or styles for the [DateTime component](https://developer.wordpress.org/block-editor/reference-guides/components/date-time/) has changed since we deployed the Events Date block. The CSS in our block works up to WP 6.1, but breaks in WP 6.2, so this PR removes the CSS. It should no longer be needed in either case.

### How to test the changes in this Pull Request:

1. On `master`, on a site running WP 6.2.*, create a new Event listing and add an Event Dates block (one should be added automatically for each new Event listing).
2. Observe that the Event Dates UI styling looks off:

<img width="818" alt="Screen Shot 2023-03-15 at 2 10 16 PM" src="https://user-images.githubusercontent.com/2230142/225432706-3d6cf947-6b66-4b36-97db-0eb9b53be97a.png">

3. Check out this branch and confirm it looks fixed now (at full-editor width):

<img width="1267" alt="Screen Shot 2023-03-15 at 2 17 57 PM" src="https://user-images.githubusercontent.com/2230142/225432859-6f688a02-9637-4024-84b9-9395378b6603.png">

4. Confirm that the block still looks okay on a site running WP 6.1.*, too. In older versions, the component won't be full-width in the editor.

### Other information:
